src/app/core/components/sobv-page-not-found/sobv-page-not-found.component.ts

```diff
@@ -6,10 +6,16 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./sobv-page-not-found.component.scss']
 })
 export class SobvPageNotFoundComponent implements OnInit {
+  error_number: string = "";
+  error_name: string = "";
```

Comment:
According to Angular styleGuide use camelCase
https://angular.io/guide/styleguide#components-as-elements

Reply:
plus